recipes/grpcio-gcp-feedstock/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "grpcio-gcp" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e292605effc7da39b7a8734c719afb12ec4b5362add3528d8afad3aa3aa9057c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --verbose"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - grpcio>=0.2.2  # must equal grpcio-gcp version
```

How about this
```suggestion
    - grpcio >={{ version }}  # must equal grpcio-gcp version
```
or do you mean:
```suggestion
    - grpcio =={{ version }}  # must equal grpcio-gcp version
```

---

`grpcio >={{ version }}` is what I meant, requirement comes from [grpc-gcp-python/src/setup.py](https://github.com/GoogleCloudPlatform/grpc-gcp-python/blob/5a2cd9807bbaf1b85402a2a364775e5b65853df6/src/setup.py#L32).  

grpcio requirement version refactored to use Jinja templating.